### PR TITLE
WIP - Invitations updates

### PR DESCRIPTION
**Pull request by saralohr**
Changes:
- Update `Checkbox`
  - no markup changes
  - Remove inner box shadows for cleaner appearance
  - Change radio to standard circle-within-a-circle appearance and add animation
  - Add hover states and change focus state (colored border)
  - Use SVG for inner icons (check and lock)
  - Add `--large` modifier
  - Add `--pill` modifier for full-width, easily tappable radios
  - Add `--colorPrimary` modifier for standard TS Blue version that is also themed

checkbox-updates

- Update `Input` and `SelectBox` to match above changes
  - Remove inner shadow
  - Change focus state to colored border (uses primary color)

input-updates

- Update `StepNav`
  - no markup changes
  - Add `--xsmall` modifier
  - Add support for numbers instead of SVG
  - Reorganize CSS to remove repetition of size properties so modifiers can easily be created and size relationships maintained

Screen Shot 2019-06-04 at 3 31 37 PM
Screen Shot 2019-06-04 at 3 31 51 PM

- Add max-width utilities that track to bp sizes
  - xs, sm, md, lg

- Add border radius utilities
  - responsive

- Add `u-spaceSidesAuto` for centering elements (`u-flexExpandSides` also does this but it's not as intuitive)

- Add assorted other utilities:
  - Position (relative and absolute)
  - Foreground and Background bg
  - u-overflowHidden
  - make all text utilities responsive

### CODE REVIEW

File: src/css/components/Checkbox.scss [hunk 1]
```diff
@@ -115,6 +120,27 @@ $Checkbox-bg-color: $cu-foreground !default;
   .Checkbox-label:before {
     border-radius: ( $Checkbox-size / 2 );
   }
+
+  .Checkbox-label:after {
+    content: '';
+    display: none;
+    height: $Checkbox-size - 4;
```
Comment (williamdingwall) [inline src/css/components/Checkbox.scss]: any reason why this couldn't be $su-xsmall?

File: src/css/components/Checkbox.scss [hunk 2]
```diff
@@ -115,6 +120,27 @@ $Checkbox-bg-color: $cu-foreground !default;
...
+    width: $Checkbox-size - 4;
+    border: $border-width-medium solid $cu-foreground;
+    border-radius: 50%;
+    position: absolute;
+    left: 2px;
```
Comment (williamdingwall) [inline src/css/components/Checkbox.scss]: and this $su-xxsmall?

File: src/css/components/Checkbox.scss [hunk 3]
```diff
@@ -131,5 +157,79 @@ $Checkbox-bg-color: $cu-foreground !default;
...
+  }
+
+  &.Checkbox--pill {
+    .Checkbox-label {
+      padding-left: $size + (2 * $su-small);
+    }
+  }
+}
+
+$Checkbox-size--large: 20px !default;
```
Comment (williamdingwall) [inline src/css/components/Checkbox.scss]: and here with $su-large

File: src/css/components/Checkbox.scss (hunk 1 above)
Comment (saralohr) [reply src/css/components/Checkbox.scss]: These changes are from an earlier commit and not part of the current diff.

**Pull request merged**